deploy/templates/route.yaml

```diff
@@ -0,0 +1,19 @@
+{{- if .Values.isOpenshift }}
+apiVersion: route.openshift.io/v1
+kind: Route
+metadata:
+  labels:
+    component: acm-cli-downloads
+  name: acm-cli-downloads
+spec:
+  host: "acm-cli-downloads.{{ .Values.ingress.domain }}"
```

Comment:
I put this back because the Openshift-generated host included the namespace as `acm-cli-downloads-default`, which I didn't really like and I needed the ingress anyway to generate the ConsoleCLIDownload links.